app/frontend/src/components/admin/AddOwner.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div>
+    This should only be done in the event that the current form owner is no
+    longer active or is out of contact in a priority event. Otherwise have the
+    current Owner or a Team Administrator for the form do this themselves.
+
+    <v-form ref="addUserForm" v-model="valid">
```

Comment:
Raw text should be wrapped in either a <p> element or similar on its own.
In regards to template nesting, I think you may be able to get away without needing a wrapping div on line 2 for the template if you did the following:
```html
<v-form ...>
  <p>This should only...</p>
  <v-row>
    <v-col>
```

Reply:
Fixed